client/pkg/pynuoadmin.py

```diff
@@ -31,52 +31,27 @@ def prereqs(self):
         return ['nuodb']
 
     def download(self):
-        # Find the latest release
-        pypi = PyPIMetadata('pynuoadmin')
-
-        self.setversion(pypi.version)
-
-        self._file = Artifact(self.name, 'pynuoadmin.tar.gz',
-                              pypi.pkgurl, chksum=pypi.pkgchksum)
-        self._file.update()
-
-        # Grab the latest version of argcomplete
-        acpypi = PyPIMetadata('argcomplete')
-        self._ac_version = acpypi.version
-        self._ac_file = Artifact('argcomplete',
-                                 'argcomplete-{}.tar.gz'.format(self._ac_version),
-                                 acpypi.pkgurl,
-                                 chksum=acpypi.pkgchksum)
-        self._ac_file.update()
+        pass
 
     def unpack(self):
         rmdir(self.pkgroot)
         mkdir(self.pkgroot)
-        unpack_file(self._file.path, self.pkgroot)
-        self.stage.basedir = os.path.join(self.pkgroot, 'pynuoadmin-{}'.format(self.stage.version))
-
-        unpack_file(self._ac_file.path, self.pkgroot)
-        self.ac_basedir = os.path.join(self.pkgroot, 'argcomplete-{}'.format(self._ac_version))
+        run(['pip', 'install', self.__PKGNAME, '-t', self.pkgroot])
 
     def install(self):
-        pdir = os.path.join('python', 'pynuoadmin')
-        self.stage.stage(pdir, ['nuodb_cli.py', 'nuodb_mgmt.py'])
-        self.stage.stage('doc', ['README.rst'])
+        self.stage.stage('python', ['./'])
 
-        acdir = os.path.join('python', 'argcomplete')
-        self.stage.stage(acdir, [os.path.join(self.ac_basedir, 'argcomplete/')])
-        self.stage.stage(acdir, [os.path.join(self.ac_basedir, 'LICENSE.rst')])
+        nuodb = self.get_package('nuodb')
+        self.stage.stage('jar', [os.path.join(nuodb.staged[0].basedir, 'jar', 'nuokeymanager.jar')])
+        self.stage.stage('python', [os.path.join(nuodb.staged[0].basedir, 'drivers', 'pynuoadmin', 'nuocmd-complete')])
 
         if Globals.target == 'lin64':
-            self.stage.stage('bin', [os.path.join(Globals.bindir, 'nuocmd')])
-            self.stage.stagefiles('etc', Globals.etcdir, ['run-java-app.sh', 'nuokeymgr'])
+            self.stage.stage('bin', [os.path.join(nuodb.staged[0].basedir, 'bin', 'nuocmd')])
+            self.stage.stagefiles('etc', os.path.join(nuodb.staged[0].basedir, 'etc'), ['run-java-app.sh', 'nuokeymgr'])
         else:
-            self.stage.stage('bin', [os.path.join(Globals.bindir, 'nuocmd.bat')])
-            self.stage.stagefiles('etc', Globals.etcdir, ['nuokeymgr.bat'])
+            self.stage.stage('bin', [os.path.join(nuodb.staged[0].basedir, 'bin', 'nuocmd.bat')])
+            self.stage.stagefiles('etc', os.path.join(nuodb.staged[0].basedir, 'etc'), ['nuokeymgr.bat'])
```

Comment:
I think we should consider adding the shell scripts into the windows package.

When I was working with Ben, he had a Windows laptop and it was super-painful to deal with, and we couldn't use Git Bash windows or WSL or anything because none of the POSIX scripting was available on Windows.

Reply:
I've been looking at adding these scripts to the Windows database package as well.